src/webgpu/gpu_test.ts

```diff
@@ -245,6 +247,23 @@ export class GPUTestSubcaseBatchState extends SubcaseBatchState {
       }
     }
   }
+
+
+  /** Skips this test case if the `langFeature` is *not* supported. */
+  requireLanguageFeatureOrSkipTestCase(langFeature: WGSLLanguageFeature) {
+    const lf = getGPU(this.recorder).wgslLanguageFeatures;
+    if (!(langFeature in lf)) {
```

Comment:
Using      lf.has(langFeature) failed with an UNIMPLEMENTED exception.
This way of doing it properly skips or does not skip.

Reply:
wgslLanguageFeatures appears to be incorrectly implemented in dawn.node.
`langFeature in lf` should always return false because that's just checking for a key with a name equal to `langFeature` in the JS object `lf`. But that's just the interface to the set, so it only has keys like `has`, etc.
`lf.has(langFeature)` is the correct way.

Also we may want to skip if `wgslLanguageFeatures` doesn't exist (to make life easier for incomplete WebGPU implementations). In this case `lf` would be `undefined`.

Reply:
Thanks. fixed now. Tested with Chrome.
At least the "unsupported" test does trigger and detect the compilation failures.